go.mod

```diff
@@ -89,4 +89,6 @@ replace github.com/lightningnetwork/lnd/clock => ./clock
 
 replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0-20180720073208-c0276d75487e
 
+replace github.com/lightninglabs/neutrino => github.com/lightninglabs/neutrino v0.11.1-0.20210428225745-0494024a599f
```

Comment:
nit: is replace still needed?

Reply:
Ah I see, this is because the PR isn't merged yet. Can you add a "depends on" remark in the description so we don't merge this too early by accident?